README.md

```diff
@@ -1,9 +1,5 @@
 <p align="center"> 
-<img src="http://swiftyspiffy.com/img/twitchlib.png" style="max-height: 300px;">
-</p>
-
-
-
+<img src="https://cdn.syzuna-programs.de/images/twitchlib.png" style="max-height: 300px;">
```

Comment:
this url is 404ing, but something syzuna can fix.

Reply:
fixed